docs/sources/review/lint-prose/rules.md

```diff
@@ -122,77 +139,151 @@ Extends: existence
 
 Use an em dash ('—') instead of '–'.
 
+_`<CURRENT TEXT>`_ was matched by one or more of the following regular expressions:
+
+- `–`
+
 [More information ->](https://developers.google.com/style/dashes)
 
 ### Grafana.GoogleGender
 
 Extends: existence
 
-Don't use '%s' as a gender-neutral pronoun.
+Don't use _`<CURRENT TEXT>`_ as a gender-neutral pronoun.
+
+_`<CURRENT TEXT>`_ was matched by one or more of the following regular expressions:
+
+- `he/she`
+- `s/he`
+- `\(s\)he`
 
 [More information ->](https://developers.google.com/style/pronouns#gender-neutral-pronouns)
 
 ### Grafana.GoogleGenderBias
 
 Extends: substitution
 
-Consider using '%s' instead of '%s'.
+Consider using _`<REPLACEMENT TEXT>`_ instead of _`<CURRENT TEXT>`_.
+
+| Current text                   | Replacement text              |
+| ------------------------------ | ----------------------------- |
+| `(?:alumnae\|alumni)`          | `graduates`                   |
+| `(?:alumna\|alumnus)`          | `graduate`                    |
+| `air(?:m[ae]n\|wom[ae]n)`      | `pilot(s)`                    |
+| `anchor(?:m[ae]n\|wom[ae]n)`   | `anchor(s)`                   |
+| `authoress`                    | `author`                      |
+| `camera(?:m[ae]n\|wom[ae]n)`   | `camera operator(s)`          |
+| `chair(?:m[ae]n\|wom[ae]n)`    | `chair(s)`                    |
+| `congress(?:m[ae]n\|wom[ae]n)` | `member(s) of congress`       |
+| `door(?:m[ae]\|wom[ae]n)`      | `concierge(s)`                |
+| `draft(?:m[ae]n\|wom[ae]n)`    | `drafter(s)`                  |
+| `fire(?:m[ae]n\|wom[ae]n)`     | `firefighter(s)`              |
+| `fisher(?:m[ae]n\|wom[ae]n)`   | `fisher(s)`                   |
+| `fresh(?:m[ae]n\|wom[ae]n)`    | `first-year student(s)`       |
+| `garbage(?:m[ae]n\|wom[ae]n)`  | `waste collector(s)`          |
+| `lady lawyer`                  | `lawyer`                      |
+| `ladylike`                     | `courteous`                   |
+| `landlord`                     | `building manager`            |
+| `mail(?:m[ae]n\|wom[ae]n)`     | `mail carriers`               |
+| `man and wife`                 | `husband and wife`            |
+| `man enough`                   | `strong enough`               |
+| `mankind`                      | `human kind`                  |
+| `manmade`                      | `manufactured`                |
+| `manpower`                     | `personnel`                   |
+| `men and girls`                | `men and women`               |
+| `middle(?:m[ae]n\|wom[ae]n)`   | `intermediary`                |
+| `news(?:m[ae]n\|wom[ae]n)`     | `journalist(s)`               |
+| `ombuds(?:man\|woman)`         | `ombuds`                      |
```

Comment:
Jut wondering... is maybe `ombudsperson` more usable here? I see it used more often in Canada as ombudsperson vs just ombuds. I doubt it'll land in the tech docs anyway... so not worth a lot of bikeshedding. :-) Just curious if it's regional.

Reply:
Oh yeah I think so too. These are some inherited rules from the Google style guide Vale implementation but we can update it.